Source/WebKit/Shared/CacheModel.h

```diff
@@ -39,7 +39,7 @@ enum class CacheModel : uint8_t {
 };
 
 void calculateMemoryCacheSizes(CacheModel, unsigned& cacheTotalCapacity, unsigned& cacheMinDeadCapacity, unsigned& cacheMaxDeadCapacity, Seconds& deadDecodedDataDeletionInterval, unsigned& backForwardCacheCapacity);
-void calculateURLCacheSizes(CacheModel, uint64_t diskFreeSize, unsigned& urlCacheMemoryCapacity, uint64_t& urlCacheDiskCapacity);
+void calculateURLCacheSizes(CacheModel, uint64_t diskFreeSize, uint64_t& urlCacheDiskCapacity);
```

Comment:
Seems like this function name no longer makes sense, now that there is only one size. Also it could use a return value instead of an out argument.

Reply:
Ok, I will rename as `calculateURLCacheDiskCapacity()`, to try to match the internal variable names, unless you have a better suggestion.